gradle/verification-metadata.xml

```diff
@@ -1437,15 +1439,11 @@
             <sha256 value="003785669f921aafe4f137468dd20a01a36111e94fd7449f26c16e7924d82d23" origin="Artifact is not signed"/>
          </artifact>
       </component>
-      <component group="org.yaml" name="snakeyaml" version="1.32">
-         <artifact name="snakeyaml-1.32.jar">
-            <pgp value="120d6f34e627ed3a772ebbfe55c7e5e701832382"/>
-         </artifact>
-      </component>
       <component group="org.ysb33r.gradle" name="grolifant" version="0.16.1">
          <artifact name="grolifant-0.16.1.jar">
             <ignored-keys>
                <ignored-key id="54ac8e2d98cfeac6" reason="also-trust requires ignoring signatures"/>
+               <ignored-key id="ea022560a81e5bd48db3d18b54ac8e2d98cfeac6" reason="PGP verification failed"/>
```

Comment:
❓ Why is it ignored?

Reply:
That looks like a bug in our generation.
The key is already ignored, with its justification just above. Only it is done with the long id and not the full fingerprint.
I'll remove the addition and replace the long id with a full fingerprint.